packages/@lwc/engine-dom/src/styles.ts

```diff
@@ -62,6 +62,7 @@ function createFreshStyleElement(content: string) {
     const elm = document.createElement('style');
     elm.type = 'text/css';
     elm.textContent = content;
+    elm.setAttribute('data-rendered-by-lwc', '');
```

Comment:
For my own education: This is a marker to distinguish between stylesheets added by us v/s all other styles brought to the page by our customers. Correct?

Reply:
Yes, I can add a comment here.

Reply:
57dd9d5a1